modules/javafx.graphics/src/test/java/test/javafx/scene/layout/CornerRadiiTest.java

```diff
@@ -53,8 +56,71 @@ public class CornerRadiiTest {
         assertFalse(r.isBottomLeftHorizontalRadiusAsPercentage());
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void negativeRadiusNotAllowed_singleConstructor() {
-        new CornerRadii(-1);
+        assertThrows(IllegalArgumentException.class, () -> new CornerRadii(-1));
+    }
+
+    @Nested
+    class IsUniformTests {
```

Comment:
just curious - what was the rationale for `@Nested` here, as the number of the test cases is fairly small.

Reply:
I will add some more tests with [JDK-8332895](https://bugs.openjdk.org/browse/JDK-8332895) (adding interpolation support), and the `@Nested` nicely groups all tests for a given functionality.